GRACE test errors on download

E.g.
```
FAILED tests/test_mlip_calculators.py::test_mlips[grace-cpu-kwargs38] - urllib.error.URLError: <urlopen error [Errno 110] Connection timed out>
```

This looks similar to the issues with ALIGNN, where I think we should just catch and skip it.

--- a/tests/test_mlip_calculators.py
+++ b/tests/test_mlip_calculators.py
@@ -3,6 +3,7 @@
 from __future__ import annotations
 
 from pathlib import Path
+from urllib.error import URLError
 from zipfile import BadZipFile
 
 import pytest
@@ -107,8 +108,11 @@ def test_mlips(arch, device, kwargs):
         calculator = choose_calculator(arch=arch, device=device, **kwargs)
         assert calculator.parameters["version"] is not None
         assert calculator.parameters["model_path"] is not None
-    except BadZipFile:
-        pytest.skip()
+    except (BadZipFile, URLError) as e:
+        if "Connection timed out" in e.msg or "File is not a zip file" in e.msg:
+            pytest.skip("Model download failed")
+        else:
+            raise e
 
 
 def test_invalid_arch():

--- a/tests/test_mlip_calculators.py
+++ b/tests/test_mlip_calculators.py
@@ -109,7 +109,7 @@ def test_mlips(arch, device, kwargs):
         assert calculator.parameters["version"] is not None
         assert calculator.parameters["model_path"] is not None
     except (BadZipFile, URLError) as e:
-        if "Connection timed out" in e.msg or "File is not a zip file" in e.msg:
+        if "Connection timed out" in e.msg or isinstance(e, BadZipFile):
             pytest.skip("Model download failed")
         else:
             raise e

--- a/tests/test_single_point.py
+++ b/tests/test_single_point.py
@@ -4,6 +4,7 @@ from __future__ import annotations
 
 from pathlib import Path
 import shutil
+from urllib.error import URLError
 
 from ase.io import read
 from numpy import isfinite
@@ -119,15 +120,21 @@ def test_extras(arch, device, expected_energy, struct, kwargs):
     """Test single point energy using extra MLIP calculators."""
     skip_extras(arch)
 
-    single_point = SinglePoint(
-        struct=DATA_PATH / struct,
-        arch=arch,
-        device=device,
-        properties="energy",
-        **kwargs,
-    )
-    energy = single_point.run()["energy"]
-    assert energy == pytest.approx(expected_energy, rel=1e-5)
+    try:
+        single_point = SinglePoint(
+            struct=DATA_PATH / struct,
+            arch=arch,
+            device=device,
+            properties="energy",
+            **kwargs,
+        )
+        energy = single_point.run()["energy"]
+        assert energy == pytest.approx(expected_energy, rel=1e-5)
+    except URLError as e:
+        if "Connection timed out" in e.msg:
+            pytest.skip("Model download failed")
+        else:
+            raise e
 
 
 def test_single_point_none():

